rabbitmq/check.py

```diff
@@ -62,6 +62,7 @@
 
 NODE_ATTRIBUTES = [
     ('fd_used', 'fd_used', float),
+    ('disk_free', 'disk_free', float),
```

Comment:
You mention `disk_free_limit` in the PR description but I see no reference to that metric. Am I missing something? 😊

Reply:
Yeah, I originally added the disk_free_limit cause I though that it shows the total disk space but it turned out that metric indicates the limit after which rabbitMQ itself will raise an alert that there's no disk space left. On my box, it was set to 50MB by default and obviously never changed. I decided not to monitor it as the benefits are not obvious.